FlowCrypt/Functionality/Services/Local Private Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -17,8 +17,7 @@ final class InMemoryPassPhraseStorage: PassPhraseStorageType {
     let passPhraseProvider: InMemoryPassPhraseProviderType
 
     init(
-        passPhraseProvider: InMemoryPassPhraseProviderType = InMemoryPassPhraseProvider.shared,
-        timeoutInSeconds: Int = 4 * 60 * 60 // 4 hours
```

Comment:
I suppose this has moved somewhere and is therefore retained?

Reply:
Yes, this change is from https://github.com/FlowCrypt/flowcrypt-ios/pull/1840 - `timeoutInSeconds` is now parsed from client configuration.